src/Controls/src/Build.Tasks/nuget/buildTransitive/netstandard2.0/Microsoft.Maui.Controls.Globs.props

```diff
@@ -150,6 +150,10 @@
       <Generator>MSBuild:Compile</Generator>
       <XamlRuntime>$(DefaultXamlRuntime)</XamlRuntime>
     </Page>
+    <Content
+      Include="$(WindowsProjectFolder)\Assets\Images\**"
```

Comment:
Why the `Images` sub folder? On android and iOS you would just stick things in the Platforms/Android/Resources/drawable (Android native convention) or Platforms/iOS/Resources (just in the folder)

For Windows, do we have any common practices that Images folder is important? What about non-image things?

I know I also did this here: https://github.com/dotnet/maui/blob/main/src/Templates/src/templates/maui-multiproject/MauiApp.1.WinUI/MauiApp.1.WinUI.csproj#L33

But, we I right? I just did that for the template. 

What happens if I add a Platforms/Windows/Assets/?

I see this also has that pattern: https://learn.microsoft.com/en-us/windows/uwp/app-resources/tailor-resources-lang-scale-contrast

But, is there a guide doc or something?